src/app/login/login.component.scss

```
@@ -402,6 +412,28 @@
+    // has a single server, so this must not depend on how many children the
+    // row actually has.
+    margin-inline-start: auto;
+  }
+
+  // Neither selector shows a hint or an error here, but mat-form-field still
+  // reserves the subscript row below the control. That makes their box taller
+  // than what they draw, so align-items: center lines up the boxes and leaves
+  // the theme toggle sitting a few px below the two fields (and pushes the row
+  // past the 76px max-height). Drop the dead space so each item's box is what
+  // it actually paints.
+  ::ng-deep mifosx-server-selector,
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)